package.json

```diff
@@ -11,15 +11,15 @@
     "url": "https://github.com/mobify/nightwatch-commands/issues"
   },
   "devDependencies": {
-    "nightwatch": "0.8.6",
+    "nightwatch": "0.9.1",
     "nodeunit": "~0.8.4",
     "grunt": "0.4.5",
     "grunt-eslint": "^17.0.0",
     "mobify-code-style": "^2.3.6"
   },
   "scripts": {
     "install": "node selenium/installation/install.js",
-    "test": "grunt lint"
+    "test": "grunt lint; grunt test"
```

Comment:
One last thing that I'd like to see changed: This `npm test` command currently uses the globally installed `grunt` command. This will fail if `grunt-cli` isn't globally installed (local machines and on circleCI). A couple ways to fix this:
- Change this line to use locally installed `grunt` with: "test": "./node_modules/.bin/grunt lint; ./node_modules/.bin/grunt test" (with "grunt-cli" as part of `package.json` deps)
- Add to `circle.yml` with a line to `npm install -g grunt-cli` to ensure it's always available on circleCI VMs